relayer/query.go

```diff
@@ -301,3 +302,36 @@ func QueryBalance(ctx context.Context, chain *Chain, address string, showDenoms
 	}
 	return out, nil
 }
+
+func QueryClientExpiration(ctx context.Context, from, to *Chain) (time.Time, error) {
```

Comment:
Nit: To align with the rest of the code base, can we change:
- `from` to `src` 
- `to` to `dst`

Sorry, I meant to note this on my first review!
Everything else looks good to me.

Reply:
Totally agree :) Changed!